frontend/webpack.config.js

```diff
@@ -17,11 +17,18 @@ module.exports = (_env, argv) => ({
     },
 
     module: {
-        rules: [{
-            test: /\.(ts|js)x?$/,
-            loader: "babel-loader",
-            ... argv.mode === "development" && { exclude: /node_modules/ },
-        }],
+        rules: [
+            {
+                test: /\.(ts|js)x?$/,
+                loader: "babel-loader",
+                ... argv.mode === "development" && { exclude: /node_modules/ },
+            },
+            {
+                test: /\.yaml$/,
+                use: "yaml-loader",
+                type: "json",
+            },
+        ],
```

Comment:
Meh, I prefer

```suggestion
        rules: [{
            test: /\.(ts|js)x?$/,
            loader: "babel-loader",
            ... argv.mode === "development" && { exclude: /node_modules/ },
        }, {
            test: /\.yaml$/,
            use: "yaml-loader",
            type: "json",
        }],
```

The extra indentation and spacing doesn't really add anything, here; it's just noise. :thinking: 

But this is bikeshedding, of course. :sweat_smile:

Reply:
Online beautifiers don't agree on that. Some prefer my formatting, some yours. I chose it because your suggestion is kind of a special case for "array of objects". Arrays of any other kind of items do not have parts of two different items in one line (like `}, {`). Dunno, I don't particularly like this. And I'd argue that whitespace can't really be "noise" as it's invisible.

Reply:
> And I'd argue that whitespace can't really be "noise" as it's invisible.

That's of course technically correct, somehow, and even though that's the best kind of correct, I disagree. xD I mean it's also technically true that every piece of code, including whitespace, is either signal or noise, and since this spacing does not signal anything useful to me, and in fact (see below) actually sends the wrong signal in some way, it's noise.

I think the pattern "array of objects" is common enough that it warrants special treatment. And when you do that, this formatting gains an additional signaling quality: what follows after `[{` is a list of objects; more often than not objects with some uniformity among them. You should be surprised if there is suddenly a `1` or `"foo"` in there, because that's probably an error.

Reply:
Dunno... nothing of what you said convinced me tbh. But you seem to have a strong opinion here, so I will reformat.

Reply:
Don't make me out to be the villain now! xD